demisto_sdk/commands/update_release_notes/tests/update_rn_test.py

```diff
@@ -556,3 +556,40 @@ def test_execute_update_invalid(self):
         update_rn = UpdateRN(pack_path="Packs/Legacy", update_type='minor', pack_files=set(),
                              added_files=set())
         update_rn.execute_update()
+
+    diff_package = [
+        ("1.0.1", "1.0.2", True),
+        ("1.0.5", "1.0.4", False),
+        ("1.0.5", "1.0.5", True)
+    ]
```

Comment:
nice UT.
can you also cover the branch of of: `len(master_metadata) > 0:`?

Reply:
In this case the `master_current_version` gets set to '99.99.99'. The UT is difficult since the mock patch function isn't allowing the variable to be redeclared within the unit test.